src/services/CertificateGenerationService.ts

```diff
@@ -78,6 +77,8 @@ class CertificateGenerationService {
       await this.generatePayload(testResult, shouldTranslateTestResult)
     );
 
+    console.log(`payload inside cert gen service: ${payload}`);
```

Comment:
same here really, what is everyone elses thoughts?

Reply:
I think this is probably okay, the payload doesn't contain any specific user info, just vehicle info

Reply:
Thinking about it, I don't think that this is needed, you can denote the object from the given vehicle which logs before this